### PR TITLE
Add index field to ApiReferences in magic items

### PR DESCRIPTION
**Pull request by Ajhaa**
## What does this do?
Adds the missing index fields to api references in magic items

## How was it tested?
Manually with local build

## Is there a Github issue this is resolving?
No

## Did you update the docs in the API? Please link an associated PR if applicable.
No

### CODE REVIEW

File: src/5e-SRD-Magic-Items.json [hunk 1]
```diff
@@ -3848,4 +4087,4 @@
     ],
     "url": "/api/magic-items/orb-of-dragonkind"
   }
-]
```
Comment (bagelbits) [inline src/5e-SRD-Magic-Items.json]: **[Nitpick]:** Can you add the newline back here?

**Review by bagelbits — COMMENTED** [review]
This looks good. One small thing, can you cut a PR to update the docs in the API? You really only need to add your change between these [two lines.](https://github.com/bagelbits/5e-srd-api/blob/master/src/views/partials/doc-resource-magic-items.ejs#L11-L12)

**Comment by Ajhaa** [comment]
https://github.com/bagelbits/5e-srd-api/pull/118

**Review by bagelbits — APPROVED** [review]
Awesome!

**Pull request merged**